libs/auth/src/openssl.ts

```diff
@@ -153,7 +153,9 @@ export function verifyFirstCertWasSignedBySecondCert(
   cert1: FilePathOrBuffer,
   cert2: FilePathOrBuffer
 ): Promise<Buffer> {
-  return openssl(['verify', '-CAfile', cert2, cert1]);
+  // -partial_chain allows for verification without a full cert chain (i.e. a chain of certs that
+  // ends with a self-signed cert)
+  return openssl(['verify', '-partial_chain', '-CAfile', cert2, cert1]);
```

Comment:
The containing helper function's name is `verifyFirstCertWasSignedBySecondCert`, so I think we're staying true to that definition with this modification.

I made this modification because the VxAdmin CA cert isn't self signed - it's signed by the VotingWorks CA. Without the `-partial_chain` flag, we'd have to provide both the VotingWorks CA cert and the VxAdmin CA cert to verify that a card VxAdmin cert was signed by the VxAdmin CA. Which is a bit redundant since we're already verifying that the VxAdmin CA cert was signed by the VotingWorks CA.